src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int limit = 100;

```

Comment:
constants should be uppercased
```suggestion
    private static final int LIMIT= 100;
```